### PR TITLE
Update user image

### LINKED ISSUE #724: Update User Image
The user image on the capture screen: https://www.sketch.com/s/a0bbeac0-767a-4873-beef-c6c7ffd5826a/a/ZOnnODG
The user image on the wallet screen: https://www.sketch.com/s/a0bbeac0-767a-4873-beef-c6c7ffd5826a/a/dljjlV9

Both these cases have copied UI and the UI does not follow the design. Please update the button to be a depth button, and refactor it so that both locations use a reusable button.

### PR DESCRIPTION
**Pull request by EngrTaofeek**
fixes #724 Updated user Image to depth button and fixed a tiny bug in the TreeCaptureScreen.kt
WhatsApp Image 2022-01-26 at 15 05 37-2
WhatsApp Image 2022-01-26 at 15 05 37

### CODE REVIEW

File: app/src/main/java/org/greenstand/android/TreeTracker/view/TextButton.kt [hunk 1]
```diff
@@ -165,6 +168,38 @@ fun DepthButtonPreview() {
     }
 }
 
+@Composable
+fun userImageButton(
```
Comment (Elforama) [inline app/src/main/java/org/greenstand/android/TreeTracker/view/TextButton.kt:172]: Please use a capitol letter at the start. UserImageButton

**Review by Elforama — CHANGES_REQUESTED** [review]
Looks good, just a minor fix.

**Comment by Elforama** [comment]
@EngrTaofeek Please resolve any conflicts and I get merge it.

**Comment by EngrTaofeek** [comment]
@Elforama conflicts fixed and branch rebased to master.

**Pull request merged**